test/jdk/tools/jpackage/helpers/jdk/jpackage/test/WindowsHelper.java

```diff
@@ -356,7 +356,7 @@ private void verifyFileAssociations(boolean exists, String suffix,
         private final String name;
     }
 
-    private static String queryRegistryValue(String keyPath, String valueName) {
+    public static String queryRegistryValue(String keyPath, String valueName) {
```

Comment:
It should be sufficient to make it package private, not public.

Reply:
Changed the modifier to package private.